CHANGELOG.md

```diff
@@ -1,331 +1,1350 @@
 # go-ipfs changelog
 
-## 0.5.0 RC1 2020-04-06
+## 0.5.0 2020-04-28
 
-**WARNING** THIS IS A DRAFT! It highlights some of the new features, along with
-a bunch of important errata. But it's _definitely_ missing a _ton_ of shiny new
-features.
+We're excited to announce go-ipfs 0.5.0! This is by far the largest go-ipfs release with ~2500 commits, 98 contributors, and over 650 PRs across ipfs, libp2p, and multiformats.
 
-### Highlights & Errata
+### Highlights
 
-This release includes many important changes users should be aware of.
+#### Content Routing
 
-#### New DHT
+The primary focus of this release was on improving content routing. That is, advertising and finding content. To that end, this release heavily focuses on improving the DHT.
 
-This release includes an almost completely rewritten DHT implementation with a
-new protocol version. From a user's perspective, providing content, finding
-content, and resolving IPNS records should simply get faster. However, this is a
-_significant_ (albeit well tested) change and significant changes are always
-risky, so heads up.
+##### Improved DHT
 
-##### Old v. New
+The distributed hash table (DHT) is how IPFS nodes keep track of who has what data. The DHT implementation has been almost completely rewritten in this release. Providing, finding content, and resolving IPNS records are now all much faster. However, there are risks involved with this update due to the significant amount of changes that have gone into this feature.
 
 The current DHT suffers from three core issues addressed in this release:
 
-1. Most peers in the DHT cannot be dialed (e.g., due to firewalls and NATs).
-   Much of a DHT query time is wasted trying to connect to peers that cannot be
-   reached.
-2. The DHT query logic doesn't properly terminate when it hits the end of the
-   query and, instead, aggressively keeps on searching.
-3. The routing tables are poorly maintained. This can cause a search that should
-   be logarithmic in the size of the network to be linear.
+- Most peers in the DHT cannot be dialed (e.g., due to firewalls and NATs). Much of a DHT query time is wasted trying to connect to peers that cannot be reached.
+- The DHT query logic doesn't properly terminate when it hits the end of the query and, instead, aggressively keeps on searching.
+- The routing tables are poorly maintained. This can cause a search that should be logarithmic in the size of the network to be linear.
```

Comment:
```suggestion
- The routing tables are poorly maintained. This can cause search performance to slow down linearly with network size, instead logarithimicaly as expected.
```